modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -60,9 +60,18 @@ def wait(*args, **kwargs):
                 partitions = result[0]
             else:
                 partitions = result
+            # When partitions have a deferred call queue, calling
+            # partition.wait() on each partition serially will serially kick
+            # off each deferred computation and wait for each partition to
+            # finish before kicking off the next one. Instead, we want to
+            # serially kick off all the deferred computations so that they can
+            # all run aynschronously, then serially wait on all the results.
```

Comment:
```suggestion
            # all run asynchronously, then wait on all the results in parallel.
```

Reply:
I wanted to clarify that while the computations are happening in parallel, we wait on them in serial (for now). What about this?

```suggestion
            # all run asynchronously in parallel, then wait serially on all the results.
```

Reply:
The sentence begins with `we want to ...` while actuaclly we do not want to wait serially and will change this in the future. Let's just not use `in parallel` here. Corrected my suggestion.